csharp/src/Microsoft.ML.OnnxRuntime/ProviderOptions.cs

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.ML.OnnxRuntime
+{
+    /// <summary>
+    /// Provider options for TensorRT.
+    /// </summary>
+    //  Example for setting:
+    //    SessionOptions.OrtTensorRTProviderOptions trt_options;
+    //    trt_options.device_id = 0;
+    //    trt_options.has_trt_options = 1;
+    //    trt_options.trt_max_workspace_size = (UIntPtr) (1<<30);
+    //    trt_options.trt_fp16_enable = 1;
+    //    trt_options.trt_int8_enable = 1;
+    //    trt_options.trt_int8_calibration_table_name = "calibration.flatbuffers";
+    //    trt_options.trt_int8_use_native_calibration_table = 0;
+    public struct OrtTensorRTProviderOptions
```

Comment:
>OrtTensorRTProviderOptions [](http://example.com/codeflow?start=18&length=26)

Here we two have two duplicate structures essentially for the same purpose. Can we unify them?